### ISSUE TITLE
Show more biomes to avoid F3 while digging up

### ISSUE THREAD

**Issue by NeunEinser** [NeunEinser/bingo#40]
Currently only mountains / emerald chunks are indicated, could be changed to show an icon per biome type

**Comment by Kri5t0fK** [comment]
How about using biome icons from Wiki? https://minecraft.fandom.com/wiki/Template:BiomeSprite?file=BiomeCSS.png
![biomeSprite](https://static.wikia.nocookie.net/minecraft_gamepedia/images/5/59/BiomeCSS.png/revision/latest?cb=20210810165413)

Image Copyright info (top right on the preview):
> This file (or parts of it) is [copyright](https://account.mojang.com/documents/minecraft_eula) [Mojang Studios](https://minecraft.fandom.com/wiki/Mojang_Studios) 
This screenshot, texture, audio, song, or other Mojang asset, or derivative of Mojang assets, does not fall under the [Fandom Terms of Use](https://www.fandom.com/terms-of-use) but is [freely usable on this wiki](https://account.mojang.com/documents/brand_guidelines).

**Comment by NeunEinser** [comment]
Yeah I thought of that, but I didn't really like these icons. Feels like a lot of them you have to look really close to actually read them, so idk.
Maybe a simple height tier list is sufficient here (just some sort of indicator telling you the general expected surface height of one biome). I don't think you really need to know the exact biome you are in in most cases.

We'll also have to see how relevant this issue still is in 1.18. Emerald chunk for example is not relevant for 1.18, as you just find them high up on the surface of mountains. I'd also not want to give information about this when you're in a cave biome.

**Comment by Kri5t0fK** [comment]
> We'll also have to see how relevant this issue still is in 1.18. (...)

True, it'll probably need rethinking a bit.
